libs/wingsdk/src/target-tf-aws/function.ts

```diff
@@ -244,7 +244,7 @@ export class Function extends cloud.Function implements IAwsFunction {
         ],
       });
       this.addNetworkConfig({
-        subnetIds: [app.subnets.private.id],
+        subnetIds: [...app.subnets.private.map((s) => s.id)],
```

Comment:
I need to write a test that shows multiple subnets attached when using `vpc = "existing"`

Reply:
Nice!